### ISSUE TITLE
Profile update banner will not dismiss

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#368]
<img width="652" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/25c3b20e-94da-4628-9662-435eb114f302">

On updating twitter handle on my profile, and clicking to update profile, the screen moved to the banner, which then wouldn't dismiss (i.e. clicking the cross had no action).

**Comment by StephDriver** [comment]
```html
<button type="button" id="close-profile-updated" class="right-6 inset-y-0 p-2">
    <img class="w-6 cursor-pointer" 
              src="/static/hourglass/media/icons/close-dark.svg" 
              alt="Close message">
</button>
```

**Comment by joemull** [comment]
Thanks for reporting this. I am able to reproduce the first part (not being able to dismiss the bubble by clicking on the X). I think it is a bug and can be handled together with https://github.com/BirkbeckCTP/hourglass/issues/354.

As for the second part:

> When it eventually did dismiss, it logged me out and took me to a log in screen, and sent me an email stating that I had "requested my email address be updated" when all I'd changed was the twitter handle.

It sounds like the "Update Email Address" button got pressed while in the course of trying to dismiss the "Profile updated" notification. They are somewhat close together on the page. Does that sound plausible? I can't think of another way this could have happened.

**Comment by StephDriver** [comment]
@joemull 
>It sounds like the "Update Email Address" button got pressed while in the course of trying to dismiss the "Profile updated" notification. They are somewhat close together on the page. Does that sound plausible? I can't think of another way this could have happened.

I think that must have happened - as I then got an email stating that I'd changed my email address and I couldn't get back in at all until I'd confirmed the email was mine.  Which does raise the question of whether we are / should be checking the email has actually changed between updates even if that button is pressed?  But a trivial issue.

**Comment by joemull** [comment]
Yes we should check that to help prevent user mistakes. Might be worth raising a separate issue.
